### PR TITLE
Make the attribute methods bs specific

### PR DESCRIPTION
**Pull request by rohit2sharma95**
Ref: https://github.com/twbs/bootstrap/pull/31827#issuecomment-728864694

### CODE REVIEW

File: js/tests/unit/dom/manipulator.spec.js [hunk 1]
```diff
@@ -73,39 +75,40 @@ describe('Manipulator', () => {
   })
 
   describe('getDataAttribute', () => {
```
Comment (Johann-S) [inline js/tests/unit/dom/manipulator.spec.js:77]: can you add a test here with a non bs attribute please ? To check it doesn't get non bs attribute

Comment (rohit2sharma95) [reply js/tests/unit/dom/manipulator.spec.js:77]: Added @Johann-S 👍

**Review by Johann-S — APPROVED** [review]
LGTM 👍 
Thanks @rohit2sharma95

**Comment by XhmikosR** [comment]
Perfect, thank you both!

**Pull request merged**